karma.conf.js

```diff
@@ -27,6 +27,11 @@ module.exports = function(config) {
     // list of files / patterns to load in the browser
     files: [
       "js/deps/jquery.js",
+      {
```

Comment:
can we merge these patterns?

Reply:
Ooops, didn’t mean to resolve that. The jQuery one gets included.

Reply:
I meant the ones below it. the `tests/` patterns.